vars/Debian.yml

```diff
@@ -1,10 +1,10 @@
 ---
 
 python_pip_packages:
-  - python-pip
+  - "{% if ansible_python_version is version_compare('3.0.0', '<') %}python-pip{% else %}python3-pip{% endif %}"
```

Comment:
Great fix. # This makes atosatto/ansible-dockerswarm compatible with ubuntu 20.04